### PR TITLE
added folder types and icons

### PR DESCRIPTION
**Pull request by Spessmen14**
Added a description for an existing unused icon (the dirty folder) and icons/code for 2 new types.

### CODE REVIEW

File: code/obj/item/pens_writing_etc.dm [hunk 1]
```diff
@@ -1148,6 +1148,21 @@
...
+	name = "filthy folder"
+	desc = "An old file folder caked with dirt and grime"
+	icon_state = "folder_dirty"
+
+/obj/item/folder/secret //For Very Secret stuff only.
+	name = "Secret folder"
+	desc = "A folder with dire warnings about unauthorized access"
+	icon_state = "folder_secret"
+
+/obj/item/folder/danger //Anything in here has contact poison that kills, incapacitates or massively debilitates if not handled with gloves!
```
Comment (TobleroneSwordfish) [inline code/obj/item/pens_writing_etc.dm:1161]: does it?

**Comment by DimWhat** [comment]
You should probably make your PR description a bit more in line to the suggested format. Also for those two new files I can give you some help with the sprites if you need it.

File: code/obj/item/pens_writing_etc.dm (hunk 1 above)
Comment (Spessmen14) [reply code/obj/item/pens_writing_etc.dm:1161]: that is merely a suggestion - the exact poison is left up to whoever actually puts one in.

**Comment by Spessmen14** [comment]
> You should probably make your PR description a bit more in line to the suggested format. Also for those two new files I can give you some help with the sprites if you need it.

What seems to be the matter with them?

File: code/obj/item/pens_writing_etc.dm (hunk 1 above)
Comment (TobleroneSwordfish) [reply code/obj/item/pens_writing_etc.dm:1161]: probably shouldn't say that it does if it doesn't

Comment (Spessmen14) [reply code/obj/item/pens_writing_etc.dm]: edited.

**Comment by ZeWaka** [comment]
> > You should probably make your PR description a bit more in line to the suggested format. Also for those two new files I can give you some help with the sprites if you need it.
> 
> What seems to be the matter with them?

there was a template provided that wasn't used

**Comment by ZeWaka** [comment]
also - i see no point in adding these if they're not going to be used anywhere

**Pull request closed without merging**